src/GT.UI/Controllers/ListingsController.cs

```diff
@@ -10,50 +12,59 @@ namespace GT.UI.Controllers
 	[ApiController]
 	public class ListingsController : ControllerBase
 	{
-		private readonly GTAppContext _context;
+		private readonly IGTListingService _ListingService;
+		private readonly UserManager<IdentityUser> _userManager;
 
-		public ListingsController(GTAppContext context)
-		{
-			_context = context;
+
+		public ListingsController(IGTListingService iGTListingService, UserManager<IdentityUser> userManager)
```

Comment:
Parameter name should be 'listingService' instead of 'iGtListingService', as it references an object of the interface type.

Reply:
Fixed spelling